tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "importHelpers": true,
+    "module": "commonjs",
+    "outDir": "lib",
+    "rootDir": "src",
+    "strict": true,
+    "target": "es2017",
```

Comment:
confirm this target against Node 8

Reply:
https://node.green/ reports similar ES2017 support for Node 8 and 10, so I'm going to leave this line as is.